### ISSUE TITLE
Code in the editor disappears randomly when Show More is clicked.

### ISSUE THREAD

**Issue by FrozenHearth** [reactjs/react.dev#5990]
The code is restored back when I click inside the editor. It happens randomly.

![reactDocsOne](https://user-images.githubusercontent.com/27160715/235936897-65d36560-eb28-48b9-9420-2a62add3e0de.PNG)
![reactDocsTwo](https://user-images.githubusercontent.com/27160715/235936910-8586d5a6-3df2-466f-b1df-0de984cc15f9.PNG)

**Comment by gaearon** [comment]
Yeah this unfortunately looks like a browser bug. It's difficult to say what triggers it exactly. If you'd like to investigate what's happening, you're welcome to dig in!

**Comment by FrozenHearth** [comment]
> Yeah this unfortunately looks like a browser bug. It's difficult to say what triggers it exactly. If you'd like to investigate what's happening, you're welcome to dig in!

After a lot of tries, I could debug it a bit.
Some of the missing code in the editor is replaced with the `<div contenteditable="false">`
Clicking inside the editor, or using inspect element sometimes seems to remove the faulty div and replace it with the correct `<div class="cm-activeLine cm-line">` element. Not sure if this is an issue with Sandpack.

![debug](https://user-images.githubusercontent.com/27160715/236051975-70f217e8-c862-42d2-8dba-5438016d6359.PNG)
![debug2](https://user-images.githubusercontent.com/27160715/236051980-32157846-bb92-4a3b-88e0-d46a54f7fc5e.PNG)
